client/src/main.rs

```diff
@@ -269,6 +269,11 @@ enum Command {
         #[clap(value_enum)]
         shell: clap_complete::Shell,
     },
+
+    ListCandidates {
+        #[clap(flatten)]
+        nat: NatOpts,
```

Comment:
The nat opts are specified when running `innernet up` and then probably persisted somewhere. It'd be best if `ListCandidates` respected whatever NatOpts are already set for the running network.

Reply:
> The nat opts are and then probably persisted somewhere.

They aren't. It is just that if you run `innernet up --daemon <args>`, the <args> are (capt obvious) retained for all the innernet fetches that the daemon does. But this is still in-memory.